tools/_setup_generation/step_rest_refman.py

```diff
@@ -100,7 +100,7 @@ def get_property_type(property_desc, from_schemas):
                 if dest_path in enterprise_paths:
                     file.write(Setup.ENTERPRISE_BANNER)
                 file.write(f"{dest_path_desc}\n\n")
-                self.navigation += f"- \"Paths for {dest_path}\": {REST_REF_DIR_PATH}/apis_{dest_path}.md\n"
+                self.navigation += f'- "Paths for {dest_path}": refmans/reference_rest/apis_{dest_path}.md\n'
```

Comment:
Can't you use the REST_REF_DIR_PATH variable? What is the issue?

Reply:
This is the error. I cannot use REST_REF_DIR_PATH as it is the absolute path to the file and the goal here is to use the relative path for the navigation